### ISSUE TITLE
Dropping at cursor

### ISSUE THREAD

**Issue by timmyg** [madebymany/sir-trevor-js#419]
For some reason, when I drop something into the ST area it always drops at the end, rather than where my cursor is. I have a feeling it is because it regenerates the <p> tags or something (when I try to take away p tags in browser, they always come back).

Any thoughts on how I can hack this? Below is a codepen linkign to my (somewhat modified) sir-trevor.js

https://codepen.io/timmyg/pen/adOPbp

**Comment by raffij** [comment]
@timmyg yeah works with native contenteditable. I'd check with a default scribe element and see what happens.

**Comment by timmyg** [comment]
@raffij any progress on this? would love to test/help

**Comment by timmyg** [comment]
@raffij do you have an expected release date on this?

**Comment by raffij** [comment]
@timmyg The fix should be on master. It's on npm as beta3. I'll have a bit of time soon to cleanup a drag / drop branch that was going to modify the behaviour of this and fix another issue you had. https://github.com/madebymany/sir-trevor-js/tree/better-drag-and-drop This will be beta4. Then a quick mobile browser round of modifications will be needed for beta5 and we should be there for the moment.

**Comment by timmyg** [comment]
@raffij awesome i will test it out next week!

**Comment by timmyg** [comment]
@raffij what is an example of a valid draggable element? Should this work on "text" block type?
